### DISCUSSION TITLE
Finalize Croissant namespace URI (with version)

### DISCUSSION THREAD

**Discussion by ccl-core** [mlcommons/croissant#417] [General]
The version of the croissant specs (format) could be captured by the namespace URL for Croissant, e.g., http://mlcommons.org/croissant/1.0

Further elements for discussion are in this doc: https://docs.google.com/document/d/1WyCE5XYImWoC-sZyonNqWfavCvFUqhPGEbqHpBuIjvk

**Comment by benjelloun** [comment]
We need to finalize the namespace for the Croissant initial release, so that all datasets can explicitly refer to it.

Proposed URI: http://mlcommons.org/croissant/1.0

Every Croissant dataset should reference this URI via the property:

dct:conformsTo: "http://mlcommons.org/croissant/1.0"

where dct = "http://purl.org/dc/terms/" is a prefix specified in the context.

Please reply on this issue if you have any objections to this proposal.

**Reply by marcenacp** [reply]
LGTM.

**Comment by marcenacp** [comment]
This is now implemented in [1.0 datasets](https://github.com/mlcommons/croissant/tree/main/datasets/1.0). Closing this discussion.
